airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java

```diff
@@ -150,6 +161,22 @@ public static Source sshWrappedSource() {
     this.stateEmissionFrequency = INTERMEDIATE_STATE_EMISSION_FREQUENCY;
   }
 
+  private ConnectorSpecification modifySpec(final ConnectorSpecification originalSpec) {
+    final ConnectorSpecification spec = Jsons.clone(originalSpec);
+    final ObjectNode properties = (ObjectNode) spec.getConnectionSpecification().get("properties");
+    ((ObjectNode) properties.get(SSL_MODE)).put("default", SSL_MODE_REQUIRE);
+
+    return spec;
+  }
+
+  @Override
+  public ConnectorSpecification spec() throws Exception {
+    if (DEPLOYMENT_MODE != null && DEPLOYMENT_MODE.equalsIgnoreCase(AdaptiveSourceRunner.CLOUD_MODE)) {
```

Comment:
FYI `if (AdaptiveSourceRunner.CLOUD_MODE.equalsIgnoreCase(DEPLOYMENT_MODE)) {` is the same thing but shorter. Same comment applies elsewhere in this diff.